homework-2/models/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Sequelize } from "sequelize";
+import { userFactory } from "./userModel";
+import { IDB } from "../interfaces/IDB";
+
+const sequelize  = new Sequelize(
+    'postgres',
```

Comment:
Bad practice to save credentials into code. It is better to use environment.
https://www.npmjs.com/package/dotenv

Reply:
Fixed, thanks